src/ui/view/dataverse/component/generalMetadata/dateInterval.tsx

```diff
@@ -0,0 +1,83 @@
+import { useTranslation } from 'react-i18next'
+import * as E from 'fp-ts/Either'
+import { pipe } from 'fp-ts/function'
+import {
+  convertToLocalizedDateIfISODateTime,
+  validateISODateRange
+} from '@/util/isoDateTime/isoDateTime'
+import { activeLanguageWithDefault } from '@/ui/languages/languages'
+
+const FormattedDateSpan = ({
+  localizedDate,
+  label
+}: {
+  label: 'from' | 'to'
+  localizedDate: string
+}): JSX.Element => {
```

Comment:
If you don't mind, I'll prefer to keep consistency for all the React components by removing the return type and declaring explicitly the `React.FC` type for the function.